tests/history-missing-bug.rs

```diff
@@ -0,0 +1,168 @@
+pub mod utils;
+
+use rgb::{interface::{AmountChange, IfaceOp}, XOutputSeal, XWitnessId};
+use utils::*;
+
+#[test]
+fn missing_history_bug() {
```

Comment:
please instead of just printing add some asserts to make the test fail. also rename this to `check_fungible_history`